docs/api-reference/test-utils/browser-test-driver.md

```diff
@@ -4,65 +4,97 @@
 
 # BrowserTestDriver (Test Automation Class)
```

Comment:
Unrelated nit: Ocular-gatsby auto extracts title from first markdown line, can we move the badge down?